packages/react/src/ComparisonTable/ComparisonTable.module.css

```diff
@@ -7,8 +7,9 @@
 
 @media screen and (min-width: 34rem) {
   .ComparisonTable--heading {
+    max-width: var(--brand-ComparisonTable-heading-maxWidth);
```

Comment:
While the token is correct, the required design token stylesheet is missing so this unfortunately won't work on its own. 

E.g. You can see the token is greyed out. It should turn blue if it's active.
<img width="523" alt="Screenshot 2023-04-20 at 23 10 24" src="https://user-images.githubusercontent.com/13340707/233498075-2cbc1720-9c96-4878-98d3-0e5e2e0279e2.png">

This is fairly easy to fix though! 

1. Go to your `ComparisonTable.tsx` file and find the line where we import the design tokens stylesheet. E.g. [here](https://github.com/primer/brand/blob/2359c0c87ec8c419ef156e8ff78eb4fcdd24b4f3/packages/react/src/ComparisonTable/ComparisonTable.tsx#L10)
2. Add another import statement, but pointing at your new stylesheet

That token should start working now. You'll also need to add `margin: 0 auto` or similar to that heading, as the max-width will force it to align left otherwise.

Reply:
I've finally managed to understand how the local development process works for this repo 😅. I used a combination of `npm run build:storybook` and `npm run start:storybook` to preview my changes. Thanks for bearing with me as I get familiar with everything! I'll make the changes now.